packages/react-hooks/src/hooks/debounced.ts

```diff
@@ -0,0 +1,29 @@
+import {useEffect, useRef, useState} from 'react';
+
+export const DELAY_COMMIT_SETTING_CHANGE = 500;
```

Comment:
No need to export this default value imo, also I would change its name for `defaultDelay` or something similar